src/main/java/minicraft/screen/Menu.java

```diff
@@ -421,6 +421,7 @@ public static class Builder {
 
 		@NotNull private RelPos titlePos = RelPos.TOP;
 		private boolean fullTitleColor = false, setTitleColor = false;
+		private boolean localizeTitle = true;
```

Comment:
The title of a menu should be a localisation entry, regardless. This is not needed.

Reply:
For an well-done menu, the title has already been localized, if the input argument of the constructor is a menu, it would not be a localization key.

Reply:
Is that meaning that the title arguments passed should have at first localized before?

Reply:
No, I'm saying the localisation key should be automatically generated. Although just remove this from the pull request.

Reply:
Automatically generated?

Reply:
Just remove it.

Reply:
Removed.